sql/moz-fx-data-shared-prod/firefox_ios/app_store_funnel_web_referrer/view.sql

```diff
@@ -0,0 +1,182 @@
+    SUM(campaign.total_downloads) AS total_downloads,
+  FROM
+    `moz-fx-data-bq-fivetran.firefox_app_store.downloads_territory_web_referrer_report` AS web_referrer
+  LEFT JOIN
+    `moz-fx-data-bq-fivetran.firefox_app_store.downloads_territory_campaign_report` AS campaign
+    USING (`date`, app_id, territory)
+  WHERE
+    web_referrer.web_referrer = "mozilla.org"
+  GROUP BY
+    `date`,
+    app_id,
+    territory,
+    web_referrer,
+    campaign
+  UNION ALL
+  SELECT
+    DATE(`date`) AS `date`,
+    app_id,
+    territory,
+    web_referrer,
+    "MISSING" AS campaign,
+    SUM(first_time_downloads) AS first_time_downloads,
+    SUM(redownloads) AS redownloads,
+    SUM(total_downloads) AS total_downloads,
+  FROM
+    `moz-fx-data-bq-fivetran.firefox_app_store.downloads_territory_web_referrer_report`
+  WHERE
+    web_referrer <> "mozilla.org"
+  GROUP BY
+    `date`,
+    app_id,
+    territory,
+    web_referrer,
+    campaign
+),
+usage_territory AS (
+  SELECT
+    DATE(web_referrer.`date`) AS `date`,
+    web_referrer.app_id,
+    web_referrer.territory,
+    web_referrer.web_referrer,
+    COALESCE(campaign.campaign, "MISSING") AS campaign,
+    SUM(campaign.active_devices) AS active_devices,
+    SUM(campaign.active_devices_last_30_days) AS active_devices_last_30_days,
+    SUM(campaign.deletions) AS deletions,
+    SUM(campaign.installations) AS installations,
+    SUM(campaign.sessions) AS sessions,
+  FROM
+    `moz-fx-data-bq-fivetran.firefox_app_store.usage_territory_web_referrer_report` AS web_referrer
+  LEFT JOIN
+    `moz-fx-data-bq-fivetran.firefox_app_store.usage_territory_campaign_report` AS campaign
+    USING (`date`, app_id, territory)
+  WHERE
+    web_referrer.web_referrer = "mozilla.org"
+  GROUP BY
+    `date`,
+    app_id,
+    territory,
+    web_referrer,
+    campaign
+  UNION ALL
+  SELECT
+    DATE(`date`) AS `date`,
+    app_id,
+    territory,
+    web_referrer,
+    "MISSING" AS campaign,
+    SUM(active_devices) AS active_devices,
+    SUM(active_devices_last_30_days) AS active_devices_last_30_days,
+    SUM(deletions) AS deletions,
+    SUM(installations) AS installations,
+    SUM(sessions) AS sessions,
+  FROM
+    `moz-fx-data-bq-fivetran.firefox_app_store.usage_territory_web_referrer_report`
+  WHERE
+    web_referrer <> "mozilla.org"
+  GROUP BY
+    `date`,
+    app_id,
+    territory,
+    web_referrer,
+    campaign
+)
+SELECT
+  `date`,
+  app_id,
+  territory,
+  web_referrer,
+  NULLIF(campaign, "MISSING") AS campaign,
+  COALESCE(impressions, 0) AS impressions,
+  COALESCE(impressions_unique_device, 0) AS impressions_unique_device,
+  COALESCE(page_views, 0) AS page_views,
+  COALESCE(page_views_unique_device, 0) AS page_views_unique_device,
+  COALESCE(first_time_downloads, 0) AS first_time_downloads,
+  COALESCE(redownloads, 0) AS redownloads,
+  COALESCE(total_downloads, 0) AS total_downloads,
+  COALESCE(active_devices, 0) AS active_devices,
```

Comment:
Doesn't this apply to all the fields? If we prepend this field with `opt_in_` should we do the same for the rest too? Otherwise, could this create confusion about the other fields? Maybe instead we should just add a comment in the table description that this table includes stats for users opt-in for data collection on the app_store?